src/deepsparse/server/tune.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import click
+
+
+@click.command(context_settings=dict(show_default=True))
+@click.argument("config_path", type=click.Path(exists=True, dir_okay=False))
```

Comment:
First iteration will optimize fields `num_cores`, `num_workers`, `endpoints[endpoint].batch_size`. Do we need to accept valid ranges for those values?